htdocs/buying.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <ul class="nav navbar-nav">
                 <li class="active"><a href="#">Buy Crypto</a></li>
-                <li><a href="/sell.html">Sell Crypto</a></li>
+                <li><a href="/selling.html">Sell Crypto</a></li>
```

Comment:
Keep this as "sell.html" -- we've already announced this link to the DASH team and they're announcing it to the community... better to revert this.